readme.md

```diff
@@ -129,7 +129,7 @@ An *empty* table with the following columns:
 |uname | Respondent's username||
 |token | Respondent's device ID (token) as supplied by Apple or Android on device registration||
 
-*Note*. This table is required to register the device IDs (tokens) in order to send push notifications, settings are stored for a single respondent on each row. Because of security issues, push notification files and configurations are excluded from this public repro. Contact the main author if you want to contribute to enhance the push notification capabilities of DearScholar.
+*Note*. This table is required to register the device IDs (tokens) in order to send push notifications, settings are stored for a single respondent on each row. Because of security issues, push notification files and configurations are excluded from this public repository. Contact the main author if you want to contribute to enhance the push notification capabilities of DearScholar.
```

Comment:
Not sure if it was supposed to be `repo` for repository, or `repro` for reproduced (as in reproduced experiment or research?)

Reply:
Oops, a newby error. It should be `repo` indeed.